reference/sql/statements/create-index.md

```diff
@@ -81,6 +81,34 @@ mysql> CREATE UNIQUE INDEX c1 ON t1 (c1);
 Query OK, 0 rows affected (0.31 sec)
 ```
 
+## Expression index
+
+TiDB can build indexes not only on one or more columns in a table, but also on an expression. When queries involve expressions, expression indexes can speed up those queries.
+
+Take the following query as an example:
+
+{{< copyable "sql" >}}
+
+```sql
+SELECT * FROM t WHERE lower(name) = "pingcap";
+```
+
+If the following expression index is established, you can use the index to speed up the query above:
+
+{{< copyable "sql" >}}
+
+```sql
+CREATE INDEX idx ON t ((lower(name)));
+```
+
+The cost of maintaining an expression index is higher than that of other indexes, because the value of the expression needs to be calculated whenever a row is inserted or updated. Since the value of the expression is already stored in the index, it does not require recalculating when the optimizer selects the expression index.
```

Comment:
```suggestion
The cost of maintaining an expression index is higher than that of maintaining other indexes, because the value of the expression needs to be calculated whenever a row is inserted or updated. The value of the expression is already stored in the index, so this value does not require recalculation when the optimizer selects the expression index.
```